src/event/CancellableTrait.php

```diff
@@ -39,7 +42,8 @@ public function isCancelled() : bool{
 		return $this->isCancelled;
 	}
 
-	public function setCancelled(bool $value = true) : void{
-		$this->isCancelled = $value;
+	public function setCancelled() : void{
+		assert(func_num_args() === 0, get_class($this) . "::setCancelled() no longer supports uncancelling");
```

Comment:
I'd rather throw an ArgumentCountError here. You know plugin devs aren't bright enough to enable assertions.

Reply:
What would you think about simply renaming it (as described in section 5)?

Reply:
Not sure. It would break 100% of cases instead of just the broken ones. A static analyser will detect this problem anyway.